recipes/Getting_Started/Getting_Started_with_Ollama.ipynb

```diff
@@ -105,7 +105,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!ollama pull granite3-dense:8b\n",
+    "!ollama pull granite3-dense:2b\n",
```

Comment:
Let's also leave a line here that mentions 8b (but commented out).

Reply:
done